lib/tty/spinner/multi.rb

```diff
@@ -17,9 +17,53 @@ class Multi
 
       def_delegators :@spinners, :each, :empty?, :length
 
-      def initialize(options = {})
-        message = options.delete(:message)
-        @options = options
+      # Initialize the indentation options and remove them from the default
+      # spinner options
+      #
+      # @api private
+      def init_indent_opts
+        @indentation_opts = {}
+        @indentation_opts[:indent] = @options.delete(:indent) { 2 }
+        @indentation_opts[:style] = @options.delete(:style) {
+          {
+            top: '',
```

Comment:
The "coverage decrease" is being cause by this line being counted as "relevant" but then not  "tested" even when there's a test covering the default case.